.gitignore

```diff
@@ -2,6 +2,7 @@
 .tox
 **/*.pyc
 /roles/geerlingguy.postgresql
+/roles/pulp.pulp_rpm_prerequisites
```

Comment:
Nice!

Reply:
@fabricio-aguiar Yeah, I recommend you symlink `ansible-pulp/roles/pulp.pulp_rpm_prerequisites` to `../../pulp_rpm_prerequisites ` on your development machine.